ProcessMaker/processmaker
# Bugfix/FOUR-9761: AI> The History is cleared before save the process

## Issue & Reproduction Steps
Steps to Reproduce 
- Have a process with some name: example "My Process"
- Create a process IN AI
- Click on USE MODEL button
- In the process name recommended use the same name as the existing process: "My Process"
- Click on save

Current Behavior
The history is cleared before creating the process if the process already exists in process table

Expected Behavior
The history should not be cleared if the process name is already taken. Only when the process is created should clear the history.

## Solution
- Emit history clear after the process is created

**Working video**

https://github.com/ProcessMaker/processmaker/assets/90727999/270ae285-9552-4976-bbc7-bfd432df771d

## How to Test
- Follow steps above. 
- First try with an existing process name, and the history should not be removed
- Then try with a non existing process name, and the history should be removed
- Verify the history was removed after the process was created.

## Related Tickets & Packages
- [FOUR-9761](https://processmaker.atlassian.net/browse/FOUR-9761)
- [package-ai PR](https://github.com/ProcessMaker/package-ai/pull/12)

## Code Review Checklist


[FOUR-9761]: https://processmaker.atlassian.net/browse/FOUR-9761?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
